src/pages/cheerpj-jnlprunner/index.astro

```diff
@@ -0,0 +1,194 @@
+---
+import { Image } from "astro:assets";
+import Shell from "../../layouts/Shell.astro";
+import Footer from "../../components/Footer.astro";
+import { DISCORD_URL } from "../../consts";
+import LinkButton from "../../components/LinkButton.astro";
+import Hero_JNLP from "../../components/jnlp-page/Hero-JNLP.astro";
+import sparkleIcon from "../../components/jnlp-page/assets/sparkle.png";
+import networkIcon from "../../components/jnlp-page/assets/network.png";
+import lockIcon from "../../components/jnlp-page/assets/lock.png";
+import billIcon from "../../components/jnlp-page/assets/bill.png";
+import layersIcon from "../../components/jnlp-page/assets/layers.png";
+import rocketIcon from "../../components/jnlp-page/assets/rocket.png";
+import screenshots from "../../components/jnlp-page/assets/screenshots.png";
+---
+
+<Shell
+	title="CheerpJ JNLP-Runner"
+	description="The browser extension for JNLP applications"
+>
+	<Hero_JNLP />
+	<div
+		class="px-4 sm:px-8 py-16 space-y-4 mx-4 flex flex-wrap items-center justify-center p-10"
+		id="features"
+	>
+		<div class="px-10 md:w-1/2 lg:w-1/3">
+			<!-- Card -->
+			<div
+				class="mb-9 rounded-xl py-8 px-7 shadow-md transition-all hover:shadow-lg sm:p-9 lg:px-6 xl:px-9"
+			>
+				<!-- Icon -->
+				<div class="mx-auto mb-7 inline-block">
+					<Image src={sparkleIcon} width="53" height="61" alt="sparkle icon" />
+				</div>
+				<!-- Text -->
+				<div>
+					<h3
+						class="mb-4 text-xl font-bold text-white sm:text-2xl lg:text-xl xl:text-2xl"
+					>
+						No Java installations
+					</h3>
+					<p class="text-base font-medium text-body-color">
+						CheerpJ JNLP-Runner is a browser extension that saves you from the
+						need of having a desktop Java installation.
+					</p>
+				</div>
+			</div>
+		</div>
```

Comment:
Make this into an actual component suite, e.g. `@/components/card/Card.astro`, `@/components/card/Icon.astro`, `@/components/card/Text.astro` or something.